### ISSUE TITLE
foo

### ISSUE THREAD

**Issue by RichCaloggero** [RichCaloggero/test#6]
## Steps to reproduce

If an iframe contains a `select` element, focusing the select and attempting to read it's options with arrow keys causes NVDA to remain silent.

### This select works fine

```
data:text/html,<select aria-label="choose one"><option>item 1</option><option>item 2</option><option>item 3</option></select>
```

### this one does not announce options as you move with arrows

```
data:text/html,<iframe src='data:text/html,<select aria-label="choose one"><option>item 1</option><option>item 2</option><option>item 3</option></select>'></iframe>
```

### Actual behavior:

NVDA does not announce the option text as you move through the select with arrows.

### Expected behavior:

NVDA reads each option's text as you move through the select with arrows.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2019.1.1

#### Windows version:
windows 10

#### Name and version of other software in use when reproducing the issue:
firefox 67.0.4

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.
